people_server/src/peopleServer.cpp

```diff
@@ -0,0 +1,65 @@
+#ifdef _WIN32
+#define NOMINMAX 1
+#endif
+
+#include <iostream>
+#include <amqpcpp.h>
+#include <nlohmann/json.hpp>
+#include <Poco/Net/NetException.h>
+#include "SimplePocoHandler.h"
+#include "processRequest.hpp"
+#include "PeopleDetector.hpp"
+
+using json = nlohmann::json;
+
+int main(){
+    const std::string cascadePath = "cascades/haarcascade_frontalface_default.xml";
+    cv::FileStorage cascadeFile(cascadePath, cv::FileStorage::READ);
+    if(!cascadeFile.isOpened()){
+        std::cerr << "could not open " << cascadePath << " file\n";
+        return -1;
+    }
+
+    const std::string address = "localhost";
+    const int port = 5672;
+    const std::string user = "myuser";
+    const std::string password = "mypassword";
+    const std::string exchange = "ImageFinder";
+    const std::string queue_name = "image_finder.people";
+    const std::string results_queue_name = "image_finder.results";
```

Comment:
This should be loaded from the ini file
Most recommended way I found is [simpleini](https://github.com/brofield/simpleini)